components/menu/__tests__/index.test.tsx

```diff
@@ -817,45 +824,11 @@ describe('Menu', () => {
     jest.useRealTimers();
   });
 
-  it('props#onOpen and props#onClose do not warn anymore', () => {
```

Comment:
Why remove this?

Reply:
I think this test case is unuseful.

Reply:
This should keep in v4 since it's v3 -> v4 compatible testing but safe to remove in v5. Ref: https://github.com/ant-design/ant-design/pull/17510/files#diff-963693cde2c2d823ca508ebce0f7de18d069ec257fc8f7da6c5888d1732ea145R639

Reply:
@zombieJ 
<img width="390" alt="image" src="https://user-images.githubusercontent.com/16132584/185530650-9f076c37-6ee0-40ca-950f-a3dc9b7847f1.png">
在ts中类型校验不会通过...不知道怎么处理了

Reply:
> @zombieJ <img alt="image" width="390" src="https://user-images.githubusercontent.com/16132584/185530650-9f076c37-6ee0-40ca-950f-a3dc9b7847f1.png"> 在ts中类型校验不会通过...不知道怎么处理了

可以写个 // @ts-ignore

Reply:
看报什么错，别轻易 ignore，可能是有问题的。